src/panels/SkyBrowserPanel/hooks.ts

```diff
@@ -0,0 +1,223 @@
+import { useEffect } from 'react';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import {
+  subscribeToSkyBrowser,
+  unsubscribeToSkyBrowser
+} from '@/redux/skybrowser/skybrowserMiddleware';
+import {
+  setActiveImage,
+  setImageCollectionData
+} from '@/redux/skybrowser/skybrowserSlice';
+
+export function useGetWwtImageCollection() {
+  const luaApi = useOpenSpaceApi();
+  const dispatch = useAppDispatch();
+  useEffect(() => {
```

Comment:
this useeffect is quite hard to read due to the length of the function.  can we maybe move function out, or otherwise simplify it somehow?